subworkflows/local/input_check.nf

```diff
@@ -17,7 +17,7 @@ workflow INPUT_CHECK {
 
     emit:
     reads                                     // channel: [ val(meta), [ reads ] ]
-    versions = SAMPLESHEET_CHECK.out.versions // channel: [ versions.yml ]
+    software_versions = SAMPLESHEET_CHECK.out.versions // channel: [ software_versions.yml ]
```

Comment:
Are you using the SAMPLESHEET_CHECK to check if the `software_versions.csv` file is valid?